R/files.R

```diff
@@ -99,6 +99,14 @@ renv_file_move <- function(source, target, overwrite = FALSE) {
   if (renv_file_exists(target))
     return(TRUE)
 
+  # On Linux try another option
+  # Use the system command to move the links
+  if(tolower(Sys.info()[["sysname"]]) == "linux"){
```

Comment:
I wonder if this should apply to all Unix-alikes? (could use `renv_platform_unix()` to check that)

Reply:
Also: please use whitespace around `if`; and for e.g. `) {`